make/compiler/Makefile.cray-prgenv

```diff
@@ -70,4 +70,13 @@ PE_CHAPEL_PKGCONFIG_LIBS := cray-pmi:cray-ugni:$(PE_CHAPEL_PKGCONFIG_LIBS)
 endif
 endif
 
+# on login/compute nodes, lustre requires the devel api to make
+# lustre/lustreapi.h available (it's implicitly available on esl nodes)
+ifeq ($(CHPL_MAKE_AUX_FILESYS), lustre)
+HAVE_LUSTRE_API_DEVEL_STATUS=$(shell pkg-config --libs cray-lustre-api-devel &>/dev/null; echo $$?)
```

Comment:
That's what I would have thought, but I didn't find a particularly elegant solution. I'm not make expert though, so I'm all ears if there's a better solution